external/swloop/swloop.m

```diff
@@ -0,0 +1,139 @@
+% Calculates the inner loop of spinw/spinwave.m in parallel
```

Comment:
I like having the equivalent matlab code documented, but should this just be as a comment in `swloop.cpp` ?

Reply:
Maybe? In order to display the `docstring` Matlab requires an `.m` file - even for mex functions, you would need an `.m` file with no code but the docstring - the main use of `swloop.m` is the doc-string (which I did not update since I prototyped it as m-code [the commented out code was the prototype before I implemented it in C++]).